scss/_buttons.scss

```diff
@@ -92,11 +92,11 @@
 
   .counter {
     color: #60b044;
-    background-color: #fff;
+    background-color: $bg-white;
   }
 
   &:hover {
-    color: #fff;
+    color: $text-white;
     background-color: darken(#60b044, 5%);
     background-image: linear-gradient(darken(#8add6d, 5%), darken(#60b044, 5%));
```

Comment:
Thought for the future, I wonder if we could get rid of gradients on buttons? It would mean deleting a bunch of CSS, give our buttons a more flatter modern look, and if we kept the border we'd still have definition to make them look "buttony".